src/localstack.js

```diff
@@ -213,14 +218,17 @@ async function getConnection ({ versionTag = 'latest', services } = {}) {
   const container = await docker.createContainer({
     HostConfig: {
       AutoRemove: true,
-      PublishAllPorts: true
+      PublishAllPorts: true,
+      Privileged: true,
```

Comment:
This needed to be enabled so localstack can execute the lambda docker image.

Reply:
We have not needed to use this in other scenarios. My understanding is that you can avoid privileged mode by mounting the docker socket in the container so that the guest container has access to the host's docker service.

Reply:
I added it while trying to find the issue with the docker socket.  Adding the Binds section is what solved my issue, I just hadn't undone the extras.  Thanks.